shared/js/contacts/import/utilities/misc.js

```diff
@@ -42,9 +42,9 @@ if (!utils.misc) {
 
       var year = normalizedDate.getFullYear();
       if (year !== FLAG_YEAR_IGNORED) {
-        options.year = 'numeric';
+        format.year = 'numeric';
```

Comment:
This could bypass the selected format isnt?
Not a blocker right now, so we can live with it.